### PR TITLE
Fixing bot shared data when its object

### PR DESCRIPTION
**Pull request by cassiodias**
* Fixing when in-depth like ${BotName.someAttrName.name}

### CODE REVIEW

File: db/helpers/botDataUtils.js [hunk 1]
```diff
@@ -190,98 +191,95 @@ function updateValues(seq, instance) {
...
+                      botId: botNames[syncBot],
+                      roomId: instance.roomId,
+                    },
+                  };
+                }
+              });
             });
+
+            // Find bot data to update
+            promises.push(BotData.findOne(whereConst));
```
Comment (BrandonMurnane) [inline db/helpers/botDataUtils.js]: why did you remove the catch and logging?

File: db/helpers/botDataUtils.js [hunk 2]
```diff
@@ -190,98 +191,95 @@ function updateValues(seq, instance) {
...
-      for (let i = ZERO; i < data.length; i += TWO) {
-        if (data[i] && data[i].value) {
-          newValue = combineValue(data[i].value, data[i + ONE]);
-          updates.push(data[i].update({ value: newValue }).catch((err) =>
-            logger.error('Shared Context error ', err)));
+        // Update bot data
+        for (let i = ZERO; i < data.length; i += TWO) {
+          if (data[i] && data[i].value) {
+            newValue = combineValue(data[i].value, data[i + ONE]);
+            updates.push(data[i].update({ value: newValue }));
```
Comment (BrandonMurnane) [inline db/helpers/botDataUtils.js]: same as above

File: db/helpers/botDataUtils.js (hunk 1 above)
Comment (cassiodias) [reply db/helpers/botDataUtils.js]: was not here, already added - it was removed by mistake

Comment (BrandonMurnane) [reply db/helpers/botDataUtils.js]: cool :)
